### PR TITLE
Fix Table of Contents

### PR DESCRIPTION
**Pull request by gutilacerda**
Hello,

I've changed how the Table of Contents is done, with @marcusfilipesr help. Unfortunately, we could not use Sphinx to build the page from the website to test the modification results. Locally, the HTML given by Sphinx does not build the links in the Table of Contents.

### CODE REVIEW

**Comment by gutilacerda** [comment]
The modifications in the first commit called "proportional damping" were removed for this pull request!

**Comment by marcusfilipesr** [comment]
Hi @gutilacerda! When you can, check the issue #914, because we can use this PR to resolve this additional update.

**Comment by gutilacerda** [comment]
Hi @marcusfilipesr. 

Ok, I will check the issue #914 and add the solution in this PR.

**Comment by gutilacerda** [comment]
Hello @marcusfilipesr. 

The correction from issue #914 is already included.

File: ross/shaft_element.py [hunk 1]
```diff
@@ -47,7 +47,8 @@ class ShaftElement(Element):
     odr : float, pint.Quantity, optional
         Outer diameter of the element at the right position (m).
         Default is equal to odl value (cylindrical element).
-    L : shaft length (m)
+    L : float
+        Shaft length (m).
```
Comment (marcusfilipesr) [inline ross/shaft_element.py:51]: Good morning @gutilacerda! In fact, the `L` parameter can be passed as a `pint.Quantity` as well, similar to `odr`. 

Additionally, as it is the first parameter in the `__init__` method, maybe it is better if it is the first in the documentation.

**Comment by marcusfilipesr** [comment]
Closes #674

File: ross/shaft_element.py (hunk 1 above)
Comment (gutilacerda) [reply ross/shaft_element.py]: Done!

File: ross/shaft_element.py [hunk 2]
```diff
@@ -36,7 +36,8 @@ class ShaftElement(Element):
 
     Parameters
     ----------
-
+    L : float, pint.Quantity
+        Shaft length (m).
```
Comment (marcusfilipesr) [inline ross/shaft_element.py:40]: One little modification left, instead of `Shaft length` it is the `Element length`.

Comment (gutilacerda) [reply ross/shaft_element.py]: Also done, @marcusfilipesr

**Comment by ross-bott** [comment]
Hi there!
I have marked this issue as stale because it has not had activity for 45 days.
Consider the following options:
- If the issue refers to a large task, break it in smaller issues that can be solved in
less than 45 days;
- Label the issue as `wontfix` or `wontfix for now` and close it.

**Comment by gutilacerda** [comment]
The tables of contents are removed now, only remaining the table on the right side.

**Review by raphaeltimbo — APPROVED** [review]

**Pull request merged**